src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
+
+const {
+  createTokenFromCookie,
+  getFlipperId,
+} = require('../../feature-toggles/helpers');
+
+describe('feature toogles', () => {
+  let oldCookie;
+
+  before(() => {
+    oldCookie = document.cookie;
+    document.cookie = '';
+  });
+
+  describe('createTokenFromCookie', () => {
+    it('should return the cookie if it already exists', () => {
+      const token = 'g84k9d0skza1zqwinl4byt';
+      document.cookie = `GACookie=${token}`;
+      expect(createTokenFromCookie('GACookie')).to.equal(
+        document.cookie.replace(
+          /(?:(?:^|.*;\s*)GACookie=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      );
+    });
+
+    it('should return a random string if the cookie does not already exist', () => {
+      document.cookie = '';
+      expect(createTokenFromCookie('GACookie')).to.not.equal(
+        document.cookie.replace(
+          /(?:(?:^|.*;\s*)GACookie=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      );
```

Comment:
I think we can simplify this too.
```suggestion
      const token = createTokenFromCookie('GACookie');
      expect(token).to.be.a('string');
      expect(token.length > 0).to.be.true;
```
We don't really care _what_ it is. Just that it's a non-empty string. Right?